README.md

```diff
@@ -147,9 +147,8 @@ require 'rr'
 
 RR is designed and tested to work against the following Ruby versions:
 
-* 2.1
-* 2.2
-* 2.3
+* 2.4
+* 2.7
```

Comment:
Could you use 2.5, 2.6 and 2.7 or remove this incomplete change?

Reply:
sorry, my mistake